README.md

```diff
@@ -1,2 +1,53 @@
-# recipes-backend
-This is a generalised backend service which can be used to deploy to multiple clusters for testing purposes
+# Recipes API backend
+
+This is a generalised backend service which can be used to deploy to multiple clusters for testing purposes with customization via the chart values.
```

Comment:
I know this is from existing repo, but reading it doesn't really make much sense

Worth re-writing whilst this is happening?
It's `deploying multiple clusters` and could probably make clearer how the kustomization is working if it's going to be getting more complex now between plum/toffee and cft/sds?